naptan_create/create_table.py

```diff
@@ -0,0 +1,62 @@
+from io import BytesIO
+import codecs
+import csv
+import boto3
+from zipfile import ZipFile
+from urllib import urlopen
+
+# dynamodb columns name
+COLUMNS = ['ATCOCode', 'NaptanCode', 'PlateCode', 'CleardownCode',
+           'CommonName', 'CommonNameLang', 'ShortCommonName', 'ShortCommonNameLang',
+           'Landmark', 'LandmarkLang', 'Street', 'StreetLang', 'Crossing',
+           'CrossingLang', 'Indicator', 'IndicatorLang', 'Bearing', 'NptgLocalityCode',
+           'LocalityName', 'ParentLocalityName', 'GrandParentLocalityName', 'Town',
+           'TownLang', 'Suburb', 'SuburbLang', 'LocalityCentre', 'GridType', 'Easting',
+           'Northing', 'Longitude', 'Latitude', 'StopType', 'BusStopType',
+           'TimingStatus', 'DefaultWaitTime', 'Notes', 'NotesLang',
+           'AdministrativeAreaCode', 'CreationDateTime', 'ModificationDateTime',
+           'RevisionNumber', 'Modification', 'Status']
+
+
+def lambda_handler():
+    url = urlopen("http://naptan.app.dft.gov.uk/DataRequest/Naptan.ashx?format=csv")
+
+    # download and unzip file, save Stops.csv in tmp file
+    try:
+        with ZipFile(BytesIO(url.read())) as my_zip_file:
+            for contained_file in my_zip_file.namelist():
+                with open("/tmp/stops_temp_file.csv", "wb") as output:
+                    if contained_file == 'Stops.csv':
```

Comment:
Let's change this to just use the `stops.csv` file you know where you extracted the file, so you should know the path to it.

Reply:
can you explain face to face? not sure I understand